apps/console/src/features/users/utils/user-management-utils.ts

```diff
@@ -28,21 +25,39 @@ export class UserManagementUtils {
      * Private constructor to avoid object instantiation from outside
      * the class.
      *
-     * @hideconstructor
      */
     // eslint-disable-next-line @typescript-eslint/no-empty-function
     private constructor() { }
 
+    /**
+     * Resolves the username of the given user.
+     *
+     * @param user - User object.
+     * @returns The username of the given user as a string.
+     */
+    public static resolveUsername = (username: string): string => {
```

Comment:
We can use this method instead https://github.com/wso2/identity-apps/blob/master/modules/core/src/helpers/profile.ts#L123

Reply:
Resolved with 8413b271b71daf184e1b29a2ed740b1b046cdfd9